dev/lib/product_taxonomy/commands/generate_release_command.rb

```diff
@@ -122,6 +122,7 @@ def get_commit_hash(ref)
     end
 
     def update_integration_mappings_stable_release
+      create_previous_version_mappings
       update_mapping_files("to_shopify.yml", "output_taxonomy", "shopify/#{@version}")
```

Comment:
If we're skipping `shopify` directories in `update_mapping_files`, this call will always do nothing, since only shopify mappings use `to_shopify.yml`.

I think there's a simpler/cleaner approach, I'll suggest it in a separate comment.